src/parser/warrior/arms/modules/spells/azeritetraits/TestOfMight.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import Enemies from 'parser/shared/modules/Enemies';
+import { formatPercentage, formatNumber } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import SPELLS from 'common/SPELLS';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+
+const RAGE_PER_PROC = 10;
+
+const testOfMightStats = traits => Object.values(traits).reduce((obj, rank) => {
+    const [strength] = calculateAzeriteEffects(SPELLS.TEST_OF_MIGHT.id, rank);
+    obj.strength += strength;
+    obj.traits += 1;
+    return obj;
+}, {
+    strength: 0,
+    traits: 0,
+});
+
+export const STAT_TRACKER = {
+    strength: combatant => {
+        return testOfMightStats(combatant.traitsBySpellId[SPELLS.TEST_OF_MIGHT.id]).strength;
+    },
+};
+
+/**
+ * Normal:
+ * When Colossus Smash expires, your Strength is increased by 29 for every 10 Rage you spent during Colossus Smash. Lasts 12 sec.
+ * 
+ * Warbreaker:
+ * When Warbreaker expires, your Strength is increased by 29 for every 10 Rage you spent during Warbreaker. Lasts 12 sec.
+ */
+
+class TestOfMight extends Analyzer {
+
+    static dependencies = {
+        enemies: Enemies,
+    };
+
+    strength = 0;
+    traits = 0;
+
+    proc = 0;
+    strengthOnToM = 0;
+    totalStrength = 0;
+    rageSpendDuringDebuff = 0;
+
+    constructor(...args) {
+        super(...args);
+        this.active = this.selectedCombatant.hasTrait(SPELLS.TEST_OF_MIGHT.id);
+        if(!this.active) {
```

Comment:
I actually can't get rid of this one because of the variable initialization underneath, otherwise i get an error: "TypeError: can't convert undefined to object".

Reply:
Fair enough